docs/index.md

```diff
@@ -6,10 +6,30 @@ A new kind of BIOS firmware that will solve the problems of ownership, security
 and compatibility allowing you create secure, efficient images that are fully
 customizable to your tastes and needs.
 
+Dasharo is a set of productized services, Open Core, and SaaS products which
```

Comment:
Dasharo is a set of productized services, Open Core, and SaaS products **which** --> **that**